### ISSUE TITLE
Steps for country deploymentd 

### ISSUE THREAD

**Issue by kreinhardt** [abyot/sun-pmt#51]

**Comment by kreinhardt** [comment]
just following up on this

**Comment by abyot** [comment]
not sure what exactly here... I mean an obvious thing to do is to just follow the standard DHIS2 installation / deployment procedure. See [here](https://ci.dhis2.org/docs/master/en/implementer/html/dhis2_implementation_guide_full.html). Chapter 4 talks about deployment strategies while chapter 8 is about server installation and configurations.

Let me know if you have particular points/issues in mind,

**Comment by kreinhardt** [comment]
ok. Is there anything that will be particular to the SUN PMT that needs to be taken into account?

**Comment by abyot** [comment]
No, there is nothing particular except those new feature developments. Using the system begins with a standard DHIS2 deployment followed by metadata creation (which you already have worked out a nice tool to guide).

One thing I can mention is that - we can make available a compiled version of the system ready for deployment. Of  course countries are also welcome to generate a WAR file - the one ready for deployment - from the git hub source.

**Comment by kreinhardt** [comment]
What would a compiled version include? Would it include the "starter kit"?

**Comment by kreinhardt** [comment]
ok. what is the difference then between the compiled version and the WAR version? Is one easier/better? What do countries usually do?

**Comment by abyot** [comment]
the compiled version is the same as the WAR. The WAR file is generated when the source code is compiled.
